### PR TITLE
adding section career 2

### PR DESCRIPTION
**Pull request by sudo-riham**
Added section in career part 2. I named the file like in Zoriana files: CareerPart2.

### CODE REVIEW

File: src/components/Home/CareerPart2.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Data from "./JobsData.json";
+import "../styles/careersection.css";
+
+const CareerPart2 = () => {
+    return(
+        <div className="div">
+            <h1 className="header">current open positions</h1>
```
Comment (MhamadR) [inline src/components/Home/CareerPart2.js:8]: You gonna be using this component in one page and as a convention, each page should have only 1 <h1> tag, so you better change this tag to any heading <h2...6> but not <h1>

File: src/components/Home/CareerPart2.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    return(
+        <div className="div">
+            <h1 className="header">current open positions</h1>
+            <h3 className="h3">Please send us an email with the application title as the subject with an attached CV in PDF format!</h3>
+            <table className="sm:justify-center ml-[2%]">
+           {
+            
+            Data && Data.map( record => {
+                return(
+                    <div className="sm:justify-center">
```
Comment (MhamadR) [inline src/components/Home/CareerPart2.js:15]: I don't think a table should contain a div block, in order to wrap your tr and td, you can use React fragment which will wrap your elements without inserting any block.

You can implement it in the following 2 ways: 
<React.Fragment>
"insert your code here"
</ React.Fragment>

or
<>
"Insert your code here"
</>

And the key={record.id} should be inside the react fragment

For more info, https://reactjs.org/docs/fragments.html

File: src/components/Home/CareerPart2.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                    <tr key={record.id}>
```
Comment (MhamadR) [inline src/components/Home/CareerPart2.js:16]: The key attribute should be in the first line of the return statement to wrap all the code. Delete it from here and check the above comment

File: src/components/styles/careersection.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
+
+.tableC{
+    margin:15%;
+}
+.header{
+    font-family: 'Poppins';
```
Comment (MhamadR) [inline src/components/styles/careersection.css:6]: This property is inherited from the index.css file, it is not necessary

File: src/components/styles/careersection.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
+
+.tableC{
+    margin:15%;
+}
+.header{
+    font-family: 'Poppins';
+    font-style: normal;
```
Comment (MhamadR) [inline src/components/styles/careersection.css:7]: The default value of this property is already "normal", it is not necessary to write it

File: src/components/styles/careersection.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
+
+.tableC{
+    margin:15%;
+}
+.header{
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 400;
```
Comment (MhamadR) [inline src/components/styles/careersection.css:8]: Also this is the default value, this line adds nothing

File: src/components/styles/careersection.css [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 50px;
+    line-height: 47px;
+    text-transform: uppercase;
+    margin:2%;
+}
+.h3{
+    font-family: 'Poppins';
```
Comment (MhamadR) [inline src/components/styles/careersection.css:15]: Same as the above comment, unnecessary property

File: src/components/styles/careersection.css [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    font-style: normal;
```
Comment (MhamadR) [inline src/components/styles/careersection.css:16]: Same as the above comment, unnecessary property

File: src/components/styles/careersection.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    font-weight: 400;
```
Comment (MhamadR) [inline src/components/styles/careersection.css:17]: Same as the above comment, unnecessary property

File: src/components/styles/careersection.css [hunk 10]
```diff
@@ -0,0 +1,21 @@
+
+.tableC{
+    margin:15%;
```
Comment (MhamadR) [inline src/components/styles/careersection.css:3]: I don't recommend to use percentage for margins or paddings. Instead you can use rem, em, and even px with breakpoints. Like px-4 sm:px-16 lg:px-[10.25rem]. 
However both career sections should have same indentation and spacing, so lets wait to merge both of them and see how we can adjust them to have same values

**Review by MhamadR — CHANGES_REQUESTED** [review]
- Changes to tag names
- Changes to style

File: src/components/styles/careersection.css (hunk 10 above)
Comment (sudo-riham) [reply src/components/styles/careersection.css:2]: alright so need for me now to edit this, right? @MhamadR

Comment (MhamadR) [reply src/components/styles/careersection.css:2]: We can keep this margin for now and change it later when we merge both your component and Zoriana's component.
This way we can make sure that both of these components are styled the same.

**Review by MhamadR — APPROVED** [review]

**Pull request merged**